parts/linux/cloud-init/artifacts/ubuntu/mellanox_mlnx_ofed.list

```diff
@@ -0,0 +1,9 @@
+#
+# Mellanox Technologies Ltd. public repository configuration file.
+# For more information, refer to http://linux.mellanox.com
+#
+# For future reference:
+# https://network.nvidia.com/support/mlnx-ofed-public-repository/
+# https://www.mellanox.com/downloads/ofed/RPM-GPG-KEY-Mellanox
+# [mlnx_ofed_24.10-1.1.4.0_base]
+deb [signed-by=/etc/apt/keyrings/mellanox_mlnx_ofed.pub] http://linux.mellanox.com/public/repo/mlnx_ofed/24.10-1.1.4.0/ubuntu24.04/arm64 ./
```

Comment:
is it safe to keep the mellanox repo present post build ? if we simply want to have the driver installed, I fear this repo /key might not work for ever so I would prefer that we remove them from the resulting VHD once the right packages are installed.

Reply:
ack